less/mixins.less

```diff
@@ -1 +1,13 @@
-// Mixins in here
\ No newline at end of file
+// Mixins in here
+.button() {
+background-color: @button-bg;
+border: none;
+color: white;
+padding: 15px 32px;
+text-align: center;
+text-decoration: none;
+display: inline-block;
+font-size: 16px;
+
+}
+
```

Comment:
- Navigation: Use the navigation.less file for all your navigation styling - DONE!
- Main Header: Use the home-page.less file for the header styling. - DONE!
- Mixin - DONE!